src/main/java/net/openhft/chronicle/bytes/Bytes.java

```diff
@@ -667,42 +788,51 @@ static VanillaBytes allocateDirect(@NotNull byte[] bytes)
     }
 
     @NotNull
-    static Bytes fromHexString(@NotNull String s) {
+    static Bytes fromHexString(@NotNull(exception = NullPointerException.class) String s) {
         return BytesInternal.fromHexString(s);
     }
 
     /**
-     * Code shared by String and StringBuffer to do searches. The
-     * source is the character array being searched, and the target
+     * Returns the lowest value such that the contents of the provided {@code source } equals the contents of the
+     * provided {@code other } starting at the provided {@code fromSourceOffset}, or -1 is returned if no such value exists.
```

Comment:
This is repeated in `@return`. I wonder if we should de-duplicate that and just put a shorter description up here,

Reply:
I could make `@return` much shorter.